fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1 +1,5 @@
-<i>LibreTube</i> is an alternative frontend for YouTube. It uses the <a href='https://github.com/TeamPiped/Piped' target='_blank' rel='nofollow'>Piped</a> API to load data and play videos and thus needs no YT account and no Google Services on your device.
+<i>LibreTube</i> is an alternative frontend for YouTube, focusing on user privacy and better watching experience. It doesn't require any YouTube account or Google Services installed on your device.
+<br />
+In comparison to YouTube, LibreTube has some additional features like SponsorBlock, DeArrow, Subscription Groups, Playlist bookmarks, and many more. Apart from that you can subscribe to channels and create private playlists just as you're used to from YouTube.
```

Comment:
```suggestion
In comparison to YouTube, LibreTube has some additional features like SponsorBlock, DeArrow, Subscription Groups, Playlist bookmarks, and many more. Apart from that, you can subscribe to channels and create private playlists just as you're used to from YouTube.
```

[Eloquent](https://flathub.org/apps/re.sonny.Eloquent) (frontend for LanguageTool) suggest a comma here.